docs/fast_data/runtime_management/control_plane_frontend.mdx

```diff
@@ -0,0 +1,191 @@
+---
+id: control_plane_frontend
+title: Control Plane Frontend
+sidebar_label: Frontend
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Runtime Management has a dedicated frontend that interacts with [Control Plane](/fast_data/runtime_management/control_plane.mdx).
+
+This frontend:
+
+- retrieves pipelines information, to show them in dedicated canvas;
+- opens web socket connections to the Control Plane, to receive updates about the state of a given pipeline;
+- sends JSON-RPC commands to the Control Plane, after the user interacts with a pipeline to pause or resume one or more data streams.  
+
+## Configuration
+
+By default, the Control Plane application exposes its frontend over the `/` base path. 
+
+This value can be overridden by using the environment `BASE_PATH` with `/<your path>/` as value.
+
+On the Endpoint section, remember to use `/<your path>` as [rewrite base path](/development_suite/api-console/api-design/endpoints.md#about-rewrite-base-path).
+
+:::tip
+Control Plane frontend can also be embedded as an iFrame, to be used for example in the [Dashboards section of Mia-Platform Console](/development_suite/monitoring/dashboard.md#managing-dashboards).
```

Comment:
We need to replace this with the statement that it can be embedded within the **runtime area** and point to the proper documentation (which is actually still wip, afaik | see https://github.com/mia-platform/documentation/pull/1320).

Reply:
Let's keep track of this and update accordingly after #1320.